internal/config/mount_config.go

```diff
@@ -31,6 +33,8 @@ type MountConfig struct {
 
 func NewMountConfig() *MountConfig {
 	mountConfig := &MountConfig{}
+	mountConfig.WriteConfig.EnableStreamingWrites = false
```

Comment:
isn't the default value false?
Also if enableStreamigWRites is not enabled by default, why are we setting it to 16. 

Lets wait for Marco to comeup with these values and then set it.

Reply:
* default value is false. I will remove line 36.

* We won't use bufferSize value until EnableStreamingWrites is enabled. 16 MB is the default chunk size of go client so I thought it should be okay to keep similar value here for start. I'll verify with Marco and update.

Reply:
If you are waiting for Marco to verify these values, dont block this PR. 

Remove the default value and proceed.

Reply:
Done